app/javascript/dashboard/constants/globals.js

```diff
@@ -61,3 +61,7 @@ export default {
   },
 };
 export const DEFAULT_REDIRECT_URL = '/app/';
+
+export const ONBOARDING_STEP_NAMES = {
+  INVITE: 'invite_team',
+};
```

Comment:
Are there any other steps we would be adding?

Reply:
We are not using any other steps as of now.